base/src/test/test_model_merge_cell_fns.rs

```diff
@@ -0,0 +1,155 @@
+#![allow(clippy::unwrap_used)]
+
+use crate::{test::util::new_empty_model, types::CellType};
+
+#[test]
+fn test_model_set_fns_related_to_merge_cells() {
+    let mut model = new_empty_model();
+
+    //creating a merge cell of D1:F2
```

Comment:
I'm sorry for doing this, but could you leave a space after the `// `? (OCD)

Reply:
No problem, corrected everywhere. yes having this automated formatter would be nice.